Resources/Prototypes/Entities/Objects/Devices/Circuitboards/Machine/production.yml

```diff
@@ -640,7 +680,6 @@
           DefaultPrototype: Beaker
           ExamineName: Glass Beaker
     - type: ReverseEngineering # Nyano
-      difficulty: 2
```

Comment:
why is a heater harder to research than a chem dispenser?

Reply:
This is a old change... I made it so they are equal level, as chem dispensers are available reserch , same with circuit printers at round start.